src/Forms/FieldList.php

```diff
@@ -837,7 +791,7 @@ public function fieldPosition($field)
     /**
      * Default template rendering of a FieldList will concatenate all FieldHolder values.
      */
-    public function forTemplate()
+    public function forTemplate(): string
```

Comment:
`FormField::FieldHolder()` has a soft return type of `DBHTMLText`

Reply:
We're concatentating in in this method though

```php
$output .= $field->FieldHolder();
```

Presumbably DBHTMLText is getting cast to string so it all works

Reply:
Ahh, sorry, my brain skipped over the concatenation